crypto/bio/b_print.c

```diff
@@ -767,7 +767,6 @@ int BIO_vprintf(BIO *bio, const char *format, va_list args)
     int ignored;
 
     dynbuf = NULL;
-    CRYPTO_push_info("doapr()");
```

Comment:
I'm not sure I understand why this is being removed -- _dopr() appears to have a codepath that allocates memory (via doapr_outch()).  On the other hand, I'm not sure I could provide an argument for why it should be present, either.